site/src/sphinx/server-annotated-service.rst

```diff
@@ -1291,3 +1291,56 @@ because they always have an empty content.
 | 304          | Not modified   |
 +--------------+----------------+
 
+Using a composite annotation
+----------------------------
+
+To avoid specifying a common set of annotations repetitively, you may want to create a composite annotation
+which is annotated by other annotations. For example, let's assume that there is a service class like the below:
+
+.. code-block:: java
+
+    public class MyAnnotatedService {
+
+        @Post("/create")
+        @ConsumesJson
+        @ProducesJson
+        @LoggingDecorator
+        @MyAuthenticationDecorator
+        public HttpResponse create() { ... }
+
+        @Post("/update")
+        @ConsumesJson
+        @ProducesJson
+        @LoggingDecorator
+        @MyAuthenticationDecorator
+        public HttpResponse update() { ... }
+    }
+
+In the above example, you had to add the same 4 annotations to the two different methods. It is obviously
+too verbose and duplicate, so we could simplify them by creating a composite annotation like the following:
+
+.. code-block:: java
+
+    @Retention(RetentionPolicy.RUNTIME)
+    @ConsumesJson
+    @ProducesJson
+    @LoggingDecorator
+    @MyAuthenticationDecorator
+    public @interface MyCreateOrUpdateApiSpec {}
+
+Now, let's rewrite the service class with the composite annotation. It is definitely less verbose than before.
+Moreover, you don't need to update both ``create()`` and ``update()`` but only ``MyCreateOrUpdateApiSpec``
+when you add more common annotations to them.
```

Comment:
How about removing these two lines? I think it's a little verbose and does not contain useful information(people will know about this without this sentence.).
It's just my opinion and if you think it needs you can leave it. :)

Reply:
Thanks to the comment! I agree with you to a certain degree. It seems a little verbose but I think it'd be better to leave it as it is because it can be used to emphasized the benefit when using a composite annotation. :-)